DESCRIPTION

```diff
@@ -74,6 +74,7 @@ Imports:
     methods,
     stringr (>= 1.2.0)
 Suggests:
+    tinytex,
```

Comment:
What about importing it and making it the default behavior?

Reply:
Yeah I was about to ask you. My only concern is that tinytex is a new package on CRAN, and I'm not sure if users with R versions lower than 3.3 are able to install it since CRAN won't provide binaries. Well, they surely can, but they just have to `install.packages('tinytex', type = 'source')` (it is a pure R package and does not require compilation).

I was wondering if we need to wait a little longer before fully routing to tinytex.